packages/website/components/SidebarLink.tsx

```diff
@@ -53,12 +53,14 @@ export function SidebarSectionButton(props: {
     <List.Item>
       <Flex
         alignItems="center"
-        justifyContent="space-between"
         className={cx([titleStyles.clickable, titleStyles.sidebarItem])}
         role="button"
         onClick={props.onClick}
       >
-        <span>{props.children}</span>
+        <Subheading marginBottom="none" marginRight="spacingXs">
+          <span>{props.children}</span>
```

Comment:
I know you didn't introduce this in your PR, but could you remove this `span` tag?
it's not doing anything and it just creates more HTML that we do not need

I removed it in inspect mode and look, nothing changes
![Screenshot 2022-01-13 at 13 37 47](https://user-images.githubusercontent.com/6597467/149331688-28507a22-0075-4980-80c7-1c35d6f50e3a.png)